src/configure/configure.ts

```diff
@@ -210,10 +210,10 @@ class PipelineConfigurer {
             }
         }
         else {
-            telemetryHelper.setTelemetry(TelemetryKeys.SourceRepoLocation, 'browse');
+            telemetryHelper.setTelemetry(TelemetryKeys.SourceRepoLocation, SourceOptions.BrowseLocalMachine);
             let selectedFolder: vscode.Uri[] = await vscode.window.showOpenDialog(
                 {
-                    openLabel: Messages.selectLabel,
+                    openLabel: Messages.selectFolderLabel,
```

Comment:
This text appears on the button in file explorer so the button appears really big . do we want this ?
![image](https://user-images.githubusercontent.com/21305620/63744744-e487f400-c8bd-11e9-8b34-340ce1c57396.png)

Reply:
Took sign off from PM. If you feel strongly against it, we can again have a discussion with him